exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -592,7 +592,21 @@ func (b *BTCMarkets) SubmitOrder(ctx context.Context, s *order.Submit) (*order.S
 	if err != nil {
 		return nil, err
 	}
-	return s.DeriveSubmitResponse(tempResp.OrderID)
+
+	submitResp, err := s.DeriveSubmitResponse(tempResp.OrderID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = submitResp.AdjustBaseAmount(tempResp.Amount)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
This is incompatible with targetAmount field:
<img width="1369" alt="image" src="https://user-images.githubusercontent.com/9261323/236085633-54ab0fdb-49e3-4716-9e81-27b8a1286671.png">

<img width="1369" alt="image" src="https://user-images.githubusercontent.com/9261323/236085773-cbaa06bf-979f-49b6-8874-2f5574c46185.png">

I would argue returning an error after an order has successfully submitted is not good. It means the order manager/CMD user won't know about it, even if it was successful